### ISSUE TITLE
Windows 10 1803 Explorer inaccessible

### ISSUE THREAD

**Issue by Timtam** [nvaccess/nvda#8239]
### Steps to reproduce:
> A list of the steps you take to demonstrate the problem. 
>
Open any explorer location (like running the execute prompt and entering C:)
NVDA won't read any files/folders, even when using up and down arrow to navigate through them.
> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
> Tell us what should happen.

### Actual behavior:
> Tell us what happens instead.

### System configuration:
NVDA version:
Version: next-15091,d89900a3

NVDA Installed

Other information:
> Example: Running in a VM

Windows version:
> Example: Windows 10 Version 1607 Build 14393.1066

Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?

Have you tried any other versions of NVDA?
> Please list them and the result
Since the recent update to Windows 10 1803, NVDA doesn't announce any folders or files in Windows Explorer anymore. Even the execution request for unsigned executables just says "Window", without being able to read the pop-ups content.
Reading through the most recent issues I thought that there were several similar issues which alreaedy got closed, so I downloaded the most recent bleeding edge from the next branch, but this one doesn't fix this issue either.
Rebooting doesn't help, NVDA will probably read the explorer window just fine about 2 or 3 times, but it stops afterwards. Killing Explorer via Taskmanager and restarting it doesn't help, nor does restarting NVDA.
Running most recent next branch bleeding edge build on Windows 10 1803, no virtual machine.
If you require some debug logs or whatever, just tell me.
Thanks

**Comment by Timtam** [comment]
Alt+Tab doesn't read properly either, btw. The first Alt+Tab entry reads fine, but all alt+tab targets afterwards won't be read by NVDA.

**Comment by josephsl** [comment]
Hi,

Yes, can you provide a log of this while NVDA is restarted with debug logging enabled? Thanks.

**Comment by josephsl** [comment]
Hi, have you tried restarting NVDA and/or your computer to see if this is fixed temporarily? CC @MichaelDCurran, looks like yet another reason for us to do something about UIA communication.

**Comment by Timtam** [comment]
Hi,

yeah, log is attached. Thats the output from the protocol viewer with logging set to debug.
Yeah, I already edited my comment above, restarting NVDA or Explorer doesn't help, restarting the PC helps for about 2 or 3 Explorer navigations, but then stops working again.
[nvda.log](https://github.com/nvaccess/nvda/files/1977757/nvda.log)

**Comment by josephsl** [comment]
Hi, can you try with add-ons disabled (specifically, Unspoken add-on turned off)? Thanks.

**Comment by Timtam** [comment]
sure, there you go, all add-ons disabled.
[nvda.log](https://github.com/nvaccess/nvda/files/1977764/nvda.log)

**Comment by zstanecic** [comment]
it is the same issue, as with the virtual audio cable, when you have 
started the multiple repeaters



W dniu 06.05.2018 o 18:03, Toni Barth pisze:

**Comment by josephsl** [comment]
Hi, hmmm, looks like this didn’t help either. Please try restarting your computer to see if this issue comes back. Thanks.

**Comment by Timtam** [comment]
OK, so I restarted. First, it looked like it worked quite fine, there were some folders (e.g. the User Folder) which needed a Alt+Tab to read properly, but it worked.
I just tried restarting NVDA to see if it still reads as it should, and there you go, just reverted to as before. Now it simply doesn't read anything anymore, just like before.
Guess the logs will be the same than before. Anything else I can help with?

**Comment by josephsl** [comment]
Hi, at this point, I think we have info we need. CC @MichaelDCurran

**Comment by Living4God1991** [comment]
Hi. I can confirm this issue in the latest Fast Ring Insider build, build
17661. Chrome has been my primary browser for a while, but I've had to
switch to Firefox for now, even though it's much slower, because neither
the file explorer or the Settings app work the way they should when it's
open. This is with both Chrome stable and Chrome Canary. I hope this helps.

**Comment by Living4God1991** [comment]
Hi. I apologize for that last message. I think I misunderstood the post at
first. However, I can still confirm that the file explorer doesn't work
right a lot of the time, particularly if Chrome is open.

**Comment by Neurrone** [comment]
Could it be the crazy number of events fired? Using accevent.exe, there are about 25 events being fired when moving from one file in explorer to another one by pressing down arrow.

**Comment by Adriani90** [comment]
I am experiencing this issue randomly on Windows 8.1 with NVDA 2017.3 as well. Here is part of the log file:

```
IO - inputCore.InputManager.executeGesture (14:29:09.217):
Input: kb(laptop):shift+delete
IO - speech.speak (14:29:09.342):
Speaking [LangChangeCommand (u'de'), u'Ordner l\xf6schen  Dialogfeld  M\xf6chten Sie diesen Ordner wirklich unwiderruflich l\xf6schen?\nDEG NORFUND authorizations.zip\nTyp: ZIP-komprimierter Ordner\nGr\xf6\xdfe: 12,8 MB\n\xc4nderungsdatum: \u200e04.\u200e06.\u200e2018 \u200f\u200e14:28']
IO - speech.speak (14:29:09.390):
Speaking [LangChangeCommand (u'de'), u'Ja  Schalter']
IO - inputCore.InputManager.executeGesture (14:29:09.592):
Input: kb(laptop):enter
DEBUGWARNING - watchdog._watcher (14:29:10.108):
Trying to recover from freeze, core stack:
  File "nvda.pyw", line 203, in <module>
  File "core.pyo", line 473, in main
  File "wx\_core.pyo", line 8657, in MainLoop
  File "wx\_core.pyo", line 7952, in MainLoop
  File "core.pyo", line 443, in Notify
  File "IAccessibleHandler.pyo", line 880, in pumpAll
  File "IAccessibleHandler.pyo", line 754, in processForegroundWinEvent
  File "IAccessibleHandler.pyo", line 521, in winEventToNVDAEvent
  File "_UIAHandler.pyo", line 321, in isUIAWindow
  File "_UIAHandler.pyo", line 315, in _isUIAWindowHelper
  File "IAccessibleHandler.pyo", line 568, in winEventCallback
  File "winUser.pyo", line 414, in getClassName

IO - speech.speak (14:29:10.546):
Speaking [LangChangeCommand (u'de'), u'\\\\pegasus\\profil$\\BOA\\Daten\\Eigene Dateien\\Downloads']
IO - speech.speak (14:29:10.654):
Speaking [LangChangeCommand (u'de'), u'Feld']
IO - inputCore.InputManager.executeGesture (14:29:11.062):
Input: kb(laptop):d
IO - inputCore.InputManager.executeGesture (14:29:11.812):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:12.046):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (14:29:12.562):
 Input: kb(laptop):rightArrow
IO - inputCore.InputManager.executeGesture (14:29:12.765):
Input: kb(laptop):leftArrow
IO - inputCore.InputManager.executeGesture (14:29:13.421):
Input: kb(laptop):alt+tab
IO - speech.speak (14:29:13.546):
Speaking [LangChangeCommand (u'de'), u'Downloads  Symbol  2 von 11']
IO - speech.speak (14:29:13.576):
Speaking [LangChangeCommand (u'de'), u'Downloads  Symbol  2 von 11']
IO - speech.speak (14:29:13.765):
Speaking [LangChangeCommand (u'de'), u'Downloads  Fenster']
IO - speech.speak (14:29:13.765):
Speaking [LangChangeCommand (u'de'), u'Elementansicht  Liste']
IO - speech.speak (14:29:13.779):
Speaking [LangChangeCommand (u'de'), u'DEG KYC AML Questionnaire for FI_BOA.docx  Nicht ausgew\xe4hlt  15 von 48']
IO - inputCore.InputManager.executeGesture (14:29:13.983):
Input: kb(laptop):alt+tab
IO - speech.speak (14:29:14.092):
Speaking [LangChangeCommand (u'de'), u'\\\\pegasus\\profil$\\BOA\\Daten\\Eigene Dateien\\Downloads  Symbol  2 von 11']
IO - speech.speak (14:29:14.124):
Speaking [LangChangeCommand (u'de'), u'\\\\pegasus\\profil$\\BOA\\Daten\\Eigene Dateien\\Downloads  Symbol  2 von 11']
IO - speech.speak (14:29:14.342):
Speaking [LangChangeCommand (u'de'), u'\\\\pegasus\\profil$\\BOA\\Daten\\Eigene Dateien\\Downloads  Fenster']
IO - speech.speak (14:29:14.342):
Speaking [LangChangeCommand (u'de'), u'Elementansicht  Liste']
IO - speech.speak (14:29:14.358):
Speaking [LangChangeCommand (u'de'), u'Diagramm in Microsoft PowerPoint.xlsx  16 von 48']
IO - inputCore.InputManager.executeGesture (14:29:14.529):
Input: kb(laptop):rightArrow
IO - inputCore.InputManager.executeGesture (14:29:14.733):
Input: kb(laptop):leftArrow
IO - inputCore.InputManager.executeGesture (14:29:14.999):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:15.296):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (14:29:15.515):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (14:29:15.687):
Input: kb(laptop):upArrow
IO - inputCore.InputManager.executeGesture (14:29:15.921):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:16.092):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:16.265):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:16.421):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (14:29:16.765):
Input: kb(laptop):alt+tab
IO - speech.speak (14:29:16.890):
Speaking [LangChangeCommand (u'de'), u'Downloads  Symbol  2 von 11']
```

**Comment by Adriani90** [comment]
As you see, it happened right after I deleted a file in Windows Explorer.

**Comment by vortex1024** [comment]
I think I also encountered this issue on someone's windows 10 1803 computer. It's strange, because mine has the same version and works fine. Jaws works, I forgot to try narrator. I wonder if there's some explorer setting triggering this. Other differences between my pc and the affected one is that I installed windows 10 pro directly, whilst the other one has updated from windows 7 to windows 10 home.

**Comment by fernando-jose-silva** [comment]
[w10.txt](https://github.com/nvaccess/nvda/files/2488417/w10.txt)
My guess is that this problem occurs in nvda when using software for a long time or by charging the use of resources in windows.
I do not have this backup software installed and I have these problems in explorer mainly after upgrading to windows 10 1809.
Attach another log.

**Comment by Timtam** [comment]
I can confirm that the latest bugfix in nvda's alphas (about to be released in 2018.4) fixes all those errors reported by me above, even with Macrium Reflect installed.
Thanks for a job well done.

**Comment by sidnc86** [comment]
@Timtam  I also installed the same backup tool yesterday to take backup of my system and this issue started occurring. Thanks for the enlightenment that Macrium Reflect is causing this issue. Well I was using free edition of Macrium. But I should tell that before Macrium Reflect, I faced the issue once in a while with NVDA and restarting NVDA used to fix it. With Macrium Reflect Free, restarting NVDA wasn't helping at all. When are we expecting NVDA 2018.4 to be released? If that is not there in recent future, can I depend on some other intermittent release?

**Comment by Timtam** [comment]
@sidnc86
glad that I was of some help. Just download and install the latest alpha (aka nightly build) of NVDA, it will fix all those issues just fine.
